bundles/org.pitest.pitclipse.ui/src/org/pitest/pitclipse/ui/view/PitView.java

```diff
@@ -29,13 +32,54 @@
  * A view displaying the HTML report generated by PIT.
  */
 public class PitView extends ViewPart implements SummaryView {
+    public final static String VIEW_ID = "org.pitest.pitclipse.ui.view.PitView";
     private Browser browser = null;
+    private PitUiUpdatePublisher publisher = null;
+    private String homeUrlString = null;
+    private Action homeAction;
+    public static String HOME_BUTTON_TEXT = "Home";
+    private Action backButton;
+    public static String BACK_BUTTON_TEXT = "<";
```

Comment:
Make it final (and move the constant to the top of the class). Also does it really have to be public?

Reply:
see above.